src/Socket/messages-recv.ts

```diff
@@ -321,6 +321,14 @@ export const makeMessagesRecvSocket = (config: SocketConfig) => {
 				msg.messageStubParameters = [ approvalMode.attrs.state ]
 			}
 
+			break
+		case 'created_membership_requests':
+			msg.messageStubType = WAMessageStubType.GROUP_MEMBERSHIP_JOIN_APPROVAL_REQUEST_NON_ADMIN_ADD
```

Comment:
i'd use `WAMessageStubType.GROUP_MEMBERSHIP_JOIN_APPROVAL_REQUEST`

Reply:
hmm

Reply:
WAMessageStubType.GROUP_MEMBERSHIP_JOIN_APPROVAL_REQUEST

is for "membership_approval_request" but it never happens